openshift/pgslice/openshift/README.md

```diff
@@ -0,0 +1,18 @@
+# pgslice partitioning image
+
+Uses https://hub.docker.com/r/ankane/pgslice to run the pgslice commands
+Runs `fill`, `analyze` and `swap` for a newly partitioned table where the original has data, to fill the partitions with existing data.
+
+## Building
+
+### Create a new build
+
+```bash
+`oc -n e1e498-tools process -f build.yaml -p VERSION=05-11-2024 | oc -n e1e498-tools apply -f -`
+```
+
+### Kick off a new build
+
+```bash
+oc -n e1e498-tools start-build pgslice --follow
+```
```

Comment:
For forgetful people like me, could we include a reminder about tagging the build as  `pglisce:prod`?

For example, I found the `Re-Tag foir production` notes in the [s-3 backup README.md](https://github.com/bcgov/wps/blob/main/openshift/s3-backup/README.md) quite helpful. Perhaps something like:

```
# tag the current production image in case we need to revert
oc -n e1e498-tools tag pgslice:prod pgslice:previous-prod
oc -n e1e498-tools tag pgslice:{the new version tag} pgslice:prod
```

Reply:
Done in: https://github.com/bcgov/wps/pull/4059/commits/00b2e61ee0e6a72f7dfd806ee7d1c96492a8e1d1

Reply:
Thank you!